### DISCUSSION TITLE
A display problem

### DISCUSSION THREAD

**Discussion by adhu2018** [community/community#111281] [Mobile]
### Select Topic Area

Bug

### Body

The screen flashes when something is displayed

https://github.com/community/community/assets/41824074/040d6b78-8a23-4e43-aadd-a8633ce104be

https://github.com/521xueweihan/HelloGitHub/blob/master/content/HelloGitHub95.md

**Comment by adhu2018** [comment]
Redmi Note 12 Turbo
Xiaomi HyperOS 1.0.2.0.UMRCNXM
Android 14
github app is 1.152.0

**Comment by MEMZ-CHROER** [comment]
Github APP can do it

**Comment by sukalaper** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> The screen flashes when something is displayed
> 
>  VID_20240308_094302.mp4 
> https://github.com/521xueweihan/HelloGitHub/blob/master/content/HelloGitHub95.md

I think "it would be better" if you used the desktop version instead of Chrome on your phone.

**Reply by adhu2018** [reply]
First of all, thank you, I just feedback the problem

**Comment by Randy0607** [comment]
Your Phone can't handle the video playback, use a laptop or PC and check if this solves the problem

**Reply by adhu2018** [reply]
The same content is displayed correctly in the browser, and I think there should be a problem with the application
